### ISSUE TITLE
Optional 'export' function

### ISSUE THREAD

**Issue by ftyers** [jonorthwash/ud-annotatrix#180]
From @olesar (and others) it would be good to have a "printable version" without the colours suitable for taking a screenshot of. Others have also asked for LaTeX (e.g. `tikz-dependency`) output too. Perhaps this could be done in a similar way to the help. E.g. you have a button for "Print" and then you can choose (via tabs?) if you want PNG (without colours) or LaTeX (or something else).

**Comment by gstark0** [comment]
@ftyers How about SVG?

**Comment by ftyers** [comment]
SVG isn't so interesting. I guess most people are going to be interested in either PNG or LaTeX.

**Comment by olesar** [comment]
I vote for SVG as well, it can be more convenient to use in web pages than png. O.14.01.2018, 14:52, "Francis Tyers" <notifications@github.com>:SVG isn't so interesting. I guess most people are going to be interested in either PNG or LaTeX.

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




-- Olga Lyashevskaya School of Linguistics, Faculty of Humanities& Linguistic Convergence LabHigher School of Economics, Moscow Dept. of Corpus Linguistics and Linguistic PoeticsVinogradov Institute of the Russian Language

**Comment by jonorthwash** [comment]
Agreed, SVG is a lot more interesting than PNG.  SVG is easily exportable to PDF and PNG; the reverse is not true.

**Comment by jonorthwash** [comment]
Duplicate of #134.
